performance-metrics/build.rs

```diff
@@ -27,10 +27,23 @@ fn main() {
         }
     }
 
+    let mut git_committer_date = "".to_string();
+    if let Ok(git_out) = Command::new("git")
+        .args(&["show", "-s", "--format=%cd"])
+        .output()
+    {
+        if git_out.status.success() {
+            if let Ok(git_out_str) = String::from_utf8(git_out.stdout) {
+                git_committer_date = git_out_str;
+            }
+        }
+    }
+
     // This println!() has a special behavior, as it will set the environment
     // variable GIT_human_readable, so that it can be reused from the binary.
     // Particularly, this is used from the main.rs to display the exact
     // version information.
     println!("cargo:rustc-env=GIT_HUMAN_READABLE={}", git_human_readable);
     println!("cargo:rustc-env=GIT_REVISION={}", git_revision);
+    println!("cargo:rustc-env=GIT_COMMITER_DATE={}", git_committer_date);
```

Comment:
I've just realised these are in the wrong place. These pull the git details for the point at which the binary is built. We want the details for the revision when the binary is run. If the binary gets rebuilt then these will be the same but if you take the binary and use it on another revision then they will be wrong.

Reply:
These are good points. It makes me wonder a similar situation: Would it be possible that  the `cloud-hypervisor` binary version does not match the repo version (given we are considering the `performance-metrics` binary version can be different from the repo version)? 

If both cases are valid, we may have to solely rely on the version information reported from the `cloud-hypervisor` binary being used for the performance metrics. If we only want to ensure the versions match when using `./dev_cli.sh --tests metrics`,  an alternative solution is always forcing a rebuild of `performance-metrics` in our scripts (considering `cargo build --all`  will ensure `cloud-hypervisor` binary is up-to-date).

Reply:
> These are good points. It makes me wonder a similar situation: Would it be possible that the `cloud-hypervisor` binary version does not match the repo version (given we are considering the `performance-metrics` binary version can be different from the repo version)?
> 

Yes, all I want you to do is move the code that looks up the commit date, revision and `git describe` to the report generation based on working directory and omit them if the repository is not git checkout.

Reply:
Alright, changes pushed as a new commit.